apps/transaction/views/transaction_export_view.py

```
@@ -0,0 +1,77 @@
+                )
+            )
+            .order_by("-paid_at", "-id")
+        )
+
+        timestamp = timezone.now()
+        filename = f"transactions-{room.slug}-{timestamp.strftime('%Y%m%d%H%M%S')}.csv"
+        response = StreamingHttpResponse(self._iter_rows(room, parents, timestamp), content_type="text/csv")
+        response["Content-Disposition"] = f'attachment; filename="{filename}"'
+        return response
+
+    def _iter_rows(self, room, parents, timestamp):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_iter_rows`

(ANN202)